atd-vzd/views/view_location_injry_count_cost_summary-schema.sql

```diff
@@ -30,15 +30,15 @@ SELECT
 	COALESCE(ccs.total_crashes, 0) + COALESCE(blueform_ccs.total_crashes, 0) AS total_crashes,
 	COALESCE(ccs.total_deaths, (0)::bigint) AS total_deaths,
 	COALESCE(ccs.total_serious_injuries, (0)::bigint) AS total_serious_injuries,
-	COALESCE(ccs.est_comp_cost, 0) + COALESCE(blueform_ccs.est_comp_cost, 0) AS est_comp_cost,
+	COALESCE(ccs.est_comp_cost, 0) + COALESCE(blueform_ccs.est_comp_cost, 0)::numeric AS est_comp_cost,
```

Comment:
Find out if the numeric cast here is redundant or document the need for this cast in the field definition of the query.

Reply:
This ensures you get a value of 0 for locations without crashes.